ptbcontrib/log_forwarder/log_forwarder.py

```diff
@@ -0,0 +1,44 @@
+import asyncio
+import logging
+from collections.abc import Iterable
+from telegram.constants import ParseMode
+from telegram.ext import ExtBot
+
+
+class LogForwarder(logging.Handler):
+    def __init__(
+        self,
+        bot: ExtBot,
+        chat_ids: Iterable[int],
+        parse_mode: ParseMode = ParseMode.MARKDOWN_V2,
+        log_levels: list[str] = ["WARN", "ERROR"],
+    ):
+        super().__init__()
+        self._bot = bot
+        self._chat_ids = chat_ids
+        self._parse_mode = parse_mode
+        self._log_levels = log_levels
+        self._loop = asyncio.get_event_loop()
+
+    def format_tg_msg(self, text: str) -> str:
+        """
+        Formats the log message to be sent to Telegram. Override this method to customize the message.
+        The default implementation applies the handler's formatter and puts the result in a Markdown code block.
+        """
+        msg = "```\n" + text + "\n```"
+        return msg
+
+    def emit(self, record: logging.LogRecord):
+        try:
+            formatted = self.format(record)
+            msg = self.format_tg_msg(formatted)
+            if record.levelname in self._log_levels:
+                for chat_id in self._chat_ids:
+                    f = self._bot.send_message(
+                        chat_id=chat_id, text=msg, parse_mode=self._parse_mode
+                    )
+                    asyncio.run_coroutine_threadsafe(f, self._loop)
+        except RecursionError:  # See issue 36272
```

Comment:
which gh repo does this issue reference?

Reply:
Python. Not the numbering in github but their own system.. Got confused when taking a look at this too

Reply:
I see. kindly add a link to it :)

Reply:
https://bugs.python.org/issue36272

Reply:
Sorry, I meant add it to the comment :D

Reply:
Yeah, figured that much, but since im not the writer of the PR i thought i may share it so that you guys dont have to look for it again 😉

Reply:
Thank you @elpekenin for linking the python issue, I put it into the comment.

Reply:
ah, I definitely did notice that @elpekenin is not the author … my bad